### PR TITLE
FIX: Sanitize filepath

### PR DESCRIPTION
**Pull request by joshicola**
* utils/args/PreFreeSurfer.py sanitizes the filepath for execution

### CODE REVIEW

File: utils/args/PreFreeSurfer.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from collections import OrderedDict
 
 from tr import tr
+from utils.gear_preliminaries import create_sanitized_filepath
```
Comment (AndysWorth) [inline utils/args/PreFreeSurfer.py:11]: I don't see this file.

**Review by AndysWorth — COMMENTED** [review]
I don't see the file utils/gear_preliminaries.py.  Am I missing something here.  Otherwise it looks good to me.

File: utils/args/PreFreeSurfer.py (hunk 1 above)
Comment (joshicola) [reply utils/args/PreFreeSurfer.py:11]: Thanks,
It is in hcp-base to be shared across gears.

Comment (joshicola) [reply utils/args/PreFreeSurfer.py:11]: https://github.com/flywheel-apps/hcp-base/pull/3

**Review by AndysWorth — APPROVED** [review]
LGTM.  I just wanted to make sure `pathvalidate` was being used.

**Pull request merged**